### ISSUE TITLE
Accessibility: autocomplete token of address title input

### ISSUE THREAD

**Issue by janobi** [shopware/platform#7012]
*PHP Version:* 8.3.17   | *Shopware Version:* 6.6.9.0  | *Affected area / extension:* Platform(Default) 

 ----

*_Actual behaviour:_*

The "title" token of the autocomplete attribute for the title input in `address-personal.html.twig` is not one of the standardized / recognized values. See: https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#autofill-detail-tokens
This causes accessibility scanners to downrate pages containing this field.

*_Expected behaviour:_*

The token should be one of the standardized ones.

*_How to reproduce:_*

Nothing to do other than comparing the template (or e.g. the registration page with the title) field against the web standard.

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-40790**



[created from NEXT-40790, comment 541106]

**Comment by shopwareBot** [comment]
[public] 

 > Priority changed to: **High**

 Thank you for your report. Based on the information in this report, we have decided to investigate this issue in more detail and are planning to provide a fix for this issue. Although we are always eager to solve such problems as soon as we can, we can't give you a more specific time frame right now. We appreciate your patience and thank you for your understanding.



[created from NEXT-40790, comment 541462]

**Comment by shopwareBot** [comment]
[public] 

 > FixVersion Change: **6.7.0.0**

 This issue will be / was solved in Shopware version ***6.7.0.0***.



[created from NEXT-40790, comment 543642]
